src/client/cli/cmd/find.cpp

```diff
@@ -19,6 +19,7 @@
 #include "common_cli.h"
 
 #include <multipass/cli/argparser.h>
+#include <multipass/cli/formatter.h>
 
 #include <iomanip>
```

Comment:
Is this now a leftover header?

Reply:
No, `chosen_formatter` is a `Formatter`.

Reply:
It looks confusing on github, but I was referring to `<iomanip>`

Reply:
d'oh, yes - both that and `<sstream>` below it. Fixed.